core/src/main/java/org/primefaces/extensions/component/inputphone/InputPhoneRenderer.java

```diff
@@ -204,13 +206,11 @@ protected void encodeScript(final FacesContext context, final InputPhone inputPh
         }
         encodeCountries(wb, "preferredCountries", inputPhone.getPreferredCountries());
 
-        if (inputPhone.isUtilsScriptRequired()) {
-            wb.attr("utilsScript",
-                        context.getApplication()
-                                    .getResourceHandler()
-                                    .createResource("inputphone/utils.js", "primefaces-extensions")
-                                    .getRequestPath());
-        }
+        wb.attr("utilsScript",
```

Comment:
As this is now always loaded, I think we can simply aggregate it into the JavaScript for this component in the pom.xml.

https://github.com/primefaces-extensions/primefaces-extensions/blob/4150d88055e475772564acf29456a4d9f68c07a0/core/pom.xml#L247-L259

Reply:
Never mind. The library needs a URL and will load it.